docs/HowTo/Transactions/Make-Transactions.md

```diff
@@ -105,7 +105,7 @@ Transaction object:
 
 | Key          | Type                | Required/Optional              | Value                                                                                  |
 |--------------|-:-:-----------------|--------------------------------|----------------------------------------------------------------------------------------|
-| **from**     | Data, 20&nbsp;bytes | Required                       | Address of the sender.                                                                 |
+| **from**     | Data, 20&nbsp;bytes | Required                       | Address of the sender. If using [multiple key files](../../Concepts/Multiple-Key-Files.md), the keystore file name must contain the sender's address.                                                                |
```

Comment:
Same as above. We don't need to mention this here.

Reply:
Done.